vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -383,12 +385,20 @@ func (r *RefCountsMap) discoverAndSync(c *client.Client, d drivers.VolumeDriver)
 			return err
 		}
 		log.Debugf("  Mounts for %v", ct.Names)
-
 		for _, mount := range containerJSONInfo.Mounts {
-			// check if the mount location belongs to vsphere plugin
-			if matchNameforVMDK(mount.Source) {
-				r.Incr(mount.Name)
-				log.Infof("  name=%v (driver=%s source=%s) (%v)",
+			// check if the mount location belongs to vmdk plugin
+			if matchNameforVMDK(mount.Source) != true {
```

Comment:
I think the comment is better than the function name. It would be nice to name  it  isVMDKMount() or something like that. `matchNameforVMDK` is confusing